enterprise/views.py

```diff
@@ -75,15 +77,21 @@ def verify_edx_resources():
     """
     Ensure that all necessary resources to render the view are present.
     """
-    required_methods = (
-        configuration_helpers,
-        get_enterprise_customer_for_request,
-        get_real_social_auth_object,
-        quarantine_session,
-        lift_quarantine
-    )
-    if any(method is None for method in required_methods):
-        raise NotConnectedToEdX(_('Methods in the Open edX platform necessary for this view are not available.'))
+    required_methods = {
+        'configuration_helpers': configuration_helpers,
+        'get_enterprise_customer_for_request': get_enterprise_customer_for_request,
+        'get_real_social_auth_object': get_real_social_auth_object,
+        'quarantine_session': quarantine_session,
+        'lift_quarantine': lift_quarantine,
+        'get_partial_pipeline': get_partial_pipeline,
+    }
```

Comment:
Nit (which means this is optional and you can ignore): instead of duplicating all the imports here, you could just change the code at the top of this file to say

```python
import_error = None

try:
    from ... import ...
except ImportError as e:
    import_error = import_error or e.message
```

Then you can just display `import_error` in the error message below:
```python
def verify_edx_resources():
    if import_error:
        raise NotConnectedToOpenEdX("An Open edX dependency required for this method is missing: {}".format(import_error))
```

Or another option is to just log a warning message within the `except ImportError` block itself, and don't bother calling `verify_edx_resources()` elsewhere.